pyOCD/target/target_kinetis.py

```diff
@@ -141,3 +164,18 @@ def massErase(self):
         else:
             logging.error("Failed to unlock. MDM status: 0x%x", val)
             return False
+
+    ## @brief Write the unsecure FSEC value to flash.
+    #
+    # This will prevent the device from being secured after the next reset.
+    def unlockFlash(self):
+        logging.info("Unlocking chip...")
+
+        logging.info("Performing mass erase")
+        self.massErase()
+
+        # Write FCF
+        logging.info("Writing FCF = %s" % repr(fcf))
+        self.flash.init()
+        self.flash.programPage(FCF_ADDR, fcf)
```

Comment:
Oh, right. I should remove this, thanks for the reminder. I did speak with the NVM team and they confirmed that you were right. ;) I was getting mass erase triggered by MDM-AP confused with the Erase All Blocks command. The Erase All Unsecure command we added was so that were is a command that works exactly like the MDM-AP mass erase.